src/android/CHIPTool/app/src/main/AndroidManifest.xml

```diff
@@ -23,6 +24,11 @@
                 <action android:name="android.intent.action.MAIN" />
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
+            <intent-filter>
+                <action android:name="android.nfc.action.NDEF_DISCOVERED" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <data android:scheme="ch" /><!-- Process CHIP URIs -->
```

Comment:
How to ensure, that other organization won't take "CH" scheme? I guess we (Zigbee Alliance) register it in IANA, right?

Reply:
I don't think Apple registered their scheme "X-HM:", so not sure CHIP will do that either, but yes, schemes can be registered in IANA according to wikipedia ;)